packages/core/system/package.json

```diff
@@ -61,6 +61,7 @@
     "@react-aria/overlays": "3.23.4",
     "@react-aria/utils": "3.26.0",
     "@react-stately/utils": "3.10.4",
-    "@react-types/datepicker": "3.8.3"
+    "@react-types/datepicker": "3.8.3",
+    "@react-types/shared": "3.25.0"
```

Comment:
Why not use the latest 3.26.0?
To avoid muti-vervisn issue like https://github.com/nextui-org/nextui/issues/4243

Reply:
because latest versions of `react-aria` are still not supported/migrated/tested

Reply:
@jrgarciadev The focus is on the mixed use of multiple versions. Although NextUI specifies a certain version, such as 3.25.0 of `@react-types/shared`, it will actually install 3.26.0 of `@react-types/shared`, and the types provided by both are not completely the same. You can try creating a new folder and initializing a project, then only install the `@nextui-org/react` package, and search the lockfile. You will see that some packages of `@react-aria/*` are using 3.26.0 of `@react-types/shared`. When imports some types from `@nextui-org/react`, such as `SortDescriptor`, for me (I use pnpm), the imported version is 3.25.0, while the component props only accept other versions, and the `SortDescriptor` does not overlap, leading to type errors.

```ts
// 3.26.0
export interface SortDescriptor {
  /** The key of the column to sort by. */
  column: Key,
  /** The direction to sort by. */
  direction: SortDirection
}

// 3.25.0
export interface SortDescriptor {
  /** The key of the column to sort by. */
  column?: Key,
  /** The direction to sort by. */
  direction?: SortDirection
}
```

Reply:
@AnYiEE just fyi. I'm currently working on it on https://github.com/nextui-org/nextui/pull/4258.